pservice/bin/ps-status.sh

```diff
@@ -14,7 +14,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PLIST=$(pgrep -u ${USER:-$(whoami)} -f pservice)
+F_SERVICEHOME="$( cd -P "$( dirname ${BASH_SOURCE[0]} )/.." && pwd )"
+source ${F_SERVICEHOME}/bin/common.sh # from ../../eservice/bin/
+
+F_USAGE='-b|--base name'
+F_BASENAME='pservice'
+
+# -----------------------------------------------------------------
+# Process command line arguments
+# -----------------------------------------------------------------
+TEMP=`getopt -o b:h --long base:,help \
+     -n 'stop.sh' -- "$@"`
```

Comment:
do you want to shut down the services if the options are wrong?

Reply:
nope, but that's also not what happens.  This is only the string which is printed out as program identifier when you provide wrong arguments. But you definite point a bug which i inherited from before but might as well also just fix ...

Reply:
just pushed fix now ...